app/services/users/delete_activity.rb

```diff
@@ -46,6 +46,11 @@ def delete_profile_info(user)
       user.credits.delete_all
       user.organization_memberships.delete_all
       user.profile_pins.delete_all
+      user.profile.update(summary: "", location: "", website_url: "", data: {})
```

Comment:
Do we want to set these to `nil` instead?  I don't know.  @jgaskins do you have opinions?

I know we've been doing a some "nullify when blank" coding.

Reply:
I decided with empty string since all profiles use empty string instead of `nil`:

https://dev.to/admin/blazer/queries/new
```
select count(*) from profiles where summary = null
#=> 0

vs.

select count(*) from profiles where summary = ''
#=> 158k
```

Reply:
> I know we've been doing a some "nullify when blank" coding.

@jeremyf AFAIK we do that to avoid treating `nil` and empty strings the same. I don't know what the right answer is here, tbh, but it might not matter since we only use profile fields for display purposes. I don't think we ever actually operate on them.

Reply:
Also, just wanted to point out, I've spoken up against using `blank?` and `present?` before and this is precisely the reason. 😄 It encourages treating `nil` and `""` (and `[]` and `{}` and…) identically, which is awesome for HTTP forms but they all actually have different semantic meanings elsewhere in the app.